src/scales/scale.linear.js

```diff
@@ -105,7 +105,14 @@ module.exports = function(Chart) {
 					var meta = chart.getDatasetMeta(datasetIndex);
 					if (chart.isDatasetVisible(datasetIndex) && IDMatches(meta)) {
 						helpers.each(dataset.data, function(rawValue, index) {
-							var value = +me.getRightValue(rawValue);
+							//	float-bar support, if y arguments are array scales will check Y value for min&max values
+							var value = 0;
+							if (helpers.isArray(rawValue)) {
```

Comment:
you will need to make this change in scale.logarithmic.js as well

Reply:
done